docs/src/language_servers.md

```diff
@@ -311,8 +311,29 @@ Follow installation instructions on [LSP-promql](https://github.com/prometheus-c
 
 ## PHP
 
+There are multiple options:
+
+### Intelephense
+
 Follow installation instructions on [LSP-intelephense](https://github.com/sublimelsp/LSP-intelephense).
 
+### Phpactor
+
+1. Install [Phpactor globally](https://phpactor.readthedocs.io/en/master/usage/standalone.html#installation-global).
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"phpactor"` client configuration to the `"clients"`:
+
+    ```json
+    {
+        "clients": {
+            "reason": {
```

Comment:
Update name?

Reply:
I'll resolve this conversation, because it's a duplicate one :)